hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/service/retransmit/KafkaRetransmissionService.java

```diff
@@ -51,7 +56,10 @@ public KafkaRetransmissionService(
                 PartitionOffset partitionOffset = new PartitionOffset(k.name(), offset, partitionId);
                 partitionOffsetList.add(partitionOffset);
                 if (!dryRun) {
-                    subscriptionOffsetChange.setSubscriptionOffset(topic.getName(), subscription, brokersClusterName, partitionOffset);
+                    multiDcExecutor.execute(new SetSubscriptionOffsetRepositoryCommand(topic.getName(), subscription,
```

Comment:
Actually, we don’t have to write info about offsets from one DC to all DCs. If offset is from DC X then we only have to write it to Zookeeper in exactly this DC X, not to other ones.